src/compliance-requirements/compliance-requirements.controller.ts

```diff
@@ -72,4 +73,50 @@ export class ComplianceRequirementsController {
     const result = await this.complianceRequirementsService.setRequirements(ticker, params);
     return handleServiceResult(result);
   }
+
+  @ApiOperation({
+    summary: 'Pause Compliance requirements for an Asset',
+    description: 'This endpoint pauses Compliance rules for an Asset.',
+  })
+  @ApiParam({
+    name: 'ticker',
+    description: 'The ticker of the Asset whose compliance requirements are to be paused',
+    type: 'string',
+    example: 'TICKER',
+  })
```

Comment:
This should get `@ApiNotFoundResponse` in the case the asset isn't found.

Reply:
There is also a permissions error if the signer doesn't own the asset. I am not sure what that will throw.

Let me know if you need help setting up the REST API to use different signers to try that out. (Make the asset with one signer, and try to pause it with another)

Reply:
The tx service can throw an error (though haven't yet wrapped my head around how it will happen for subscribe on ws) - so added a swagger decorator to combine the possible error responses

Reply:
> though haven't yet wrapped my head around how it will happen for subscribe on ws

I am assuming you mean the subscription part as in when `webhookUrl` is passed and `ws` is for `websocket`? 

In that case we still run the sdk transaction validation layer (e.g. 'prepareAsset') and any errors like that will be handled as a non subscribed version. (hopefully this catches most issues, since the user will still pay gas fees if the SDK missed something)

If the SDK missed a chain constraint in the validation step (or the state changes between check and submission) the "async" error will need to be posted to the webhook subscriber, although we'd have returned 2xx to the submitter. 

In this service the websocket is an implementation detail of communicating with a  chain node, where as the SubscriptionService here uses regular HTTP `POST` for the public API.